src/Symfony/Component/Security/Guard/Token/GuardTokenInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Symfony\Component\Security\Guard\Token;
+
+/**
+ * An empty interface that both guard tokens implement.
+ *
+ * This interface is used by the GuardAuthenticationProvider to know
+ * that a token belongs to its system.
```

Comment:
An interface that knows about its implementations? ;)

What about replacing `An empty interface` by `A marker interface`?